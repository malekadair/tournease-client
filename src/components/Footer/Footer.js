import React, { Component } from "react";

class Footer extends Component {
  // lol, nope
  render() {
    return <div>Footer</div>;
  }
}

export default Footer;
