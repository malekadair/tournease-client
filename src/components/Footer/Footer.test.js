import React from "react";
import toJson from "enzyme-to-json";
import Footer from "./Footer";

describe(`Footer component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a .Footer by default", () => {
    const wrapper = <Footer />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the Footer given props", () => {
    const wrapper = <Footer {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
