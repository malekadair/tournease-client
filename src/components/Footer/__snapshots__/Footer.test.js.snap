// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders a .Footer by default 1`] = `undefined`;

exports[`Footer component renders the Footer given props 1`] = `undefined`;
